### PR TITLE
change the sdn pod to run the iptables command on the host

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
In order to use the hosts versions of the iptables command scripts are added to replace the containerized versions to point to the versions in the host.

https://github.com/openshift/cluster-network-operator/pull/127 needs to merge first so the hosts iptables will be mounted in

### CODE REVIEW

File: images/sdn/Dockerfile.rhel [hunk 1]
```diff
@@ -25,4 +27,4 @@ RUN INSTALL_PKGS=" \
 LABEL io.k8s.display-name="OpenShift SDN" \
       io.k8s.description="This is a component of OpenShift and contains the networking tool stack for the default SDN implementation." \
       io.openshift.tags="openshift,sdn"
-ENTRYPOINT [ "/usr/local/bin/openshift-sdn" ]
+ENTRYPOINT [ "/usr/bin/openshift-sdn" ]
```
Comment (sdodson) [inline images/sdn/Dockerfile.rhel]: Was this change intentional? Doesn't seem related to other changes.

Comment (JacobTanenbaum) [reply images/sdn/Dockerfile.rhel]: It was intentional, if you look further up in the dockerfile you can see that openshift-sdn was put in usr/bin and /usr/local/bin. Would it be better to separate that into its own commit?

Comment (sdodson) [reply images/sdn/Dockerfile.rhel]: Sure, go ahead and separate it into a unique commit so there's some context around that specific change. Is the ENTRYPOINT used or overridden elsewhere? I'm confused as to how it would've been working prior to your change.

Comment (JacobTanenbaum) [reply images/sdn/Dockerfile.rhel]: RHEL 7 has unified /usr/bin and /bin so functionally they are the same but it is good practice to refer to the binary as in the same location all the time. I submitted a PR to the operator to do the same.

https://github.com/openshift/cluster-network-operator/pull/128

**Comment by knobunc** [comment]
/override ci/prow/unit
(The test has passed, not sure why the bot has not picked it up)

**Comment by knobunc** [comment]
@stevekuznetsov any idea why the test has passed but the bot thinks it is pending?  How can I re-synchronize the state without bugging you in the future?
